pages/teams/[name].js

```diff
@@ -243,7 +246,31 @@ class TeamDashboard extends React.Component {
         )
         actions.push(deleteAction)
       }
-      actions.push(<Tag color="#5cdbd3">{status}</Tag>)
+
+      if (cluster.status.components) {
```

Comment:
Is `status` in `cluster.status` a component?

If it is, would it make sense for the `cluster status` component to render itself? i.e. the `<Popover>` tag can be the status's markup.

Reply:
`cluster.status.status` is just a string in the JSON that describes a cluster, it looks like this
```
{
    "status": {
        "adminToken": {
            "metadata": {
                "creationTimestamp": null
            }
        },
        "caCertificate": "",
        "components": [
            {
                "name": "provision",
                "status": "Success",
                "message": "cluster has been successfully provisioned"
            },
            {
                "name": "api",
                "status": "Success",
                "message": "api service proxy is running and available"
            },
            {
                "name": "inherited_roles",
                "status": "Success",
                "message": "provision one of more inherited users on cluster"
            },
            {
                "name": "applications",
                "status": "Success",
                "message": "in cluster applications installed"
            }
        ],
        "endpoint": "35.246.14.235",
        "apiEndpoint": "https://34.89.110.119",
        "status": "Success"
    }
}
```

Reply:
Gotcha. Then pls ignore :-)